cmds/exp/ipmidump/ipmidump.go

```diff
@@ -194,3 +207,112 @@ func selInfo() {
 		}
 	}
 }
+
+func lanConfig() {
+	const (
+		setInProgress byte = iota
+		_
+		_
+		IPAddress
+		IPAddressSrc
+		MACAddress
+	)
+
+	setInProgressStr := []string{
+		"Set Complete", "Set In Progress", "Commit Write", "Reserved",
+	}
+
+	IPAddressSrcStr := []string{
+		"Unspecified", "Static Address", "DHCP Address", "BIOS Assigned Address",
+	}
+
+	ipmi, err := ipmi.Open(0)
+	if err != nil {
+		fmt.Printf("Failed to open ipmi device: %v\n", err)
+	}
+	defer ipmi.Close()
+
+	// data 1	completion code
+	// data 2	parameter revision, 0x11
+	// data 3:N	data
+
+	// set in progress
+	if buf, err := ipmi.GetLanConfig(1, setInProgress); err != nil {
+		fmt.Printf("Failed to get LAN config: %v\n", err)
+	} else {
+		fmt.Printf("Set In Porgress   : ")
+		if int(buf[2]) < len(setInProgressStr) {
+			fmt.Println(setInProgressStr[buf[2]])
+		} else {
+			fmt.Println("Unknown")
+			fmt.Printf("%v\n", buf)
+		}
+	}
+
+	// ip address source
+	if buf, err := ipmi.GetLanConfig(1, IPAddressSrc); err != nil {
+		fmt.Printf("Failed to get LAN config: %v\n", err)
+	} else {
+		fmt.Printf("IP Address Source : ")
+		if int(buf[2]) < len(IPAddressSrcStr) {
+			fmt.Println(IPAddressSrcStr[buf[2]])
+		} else {
+			fmt.Println("Other")
+			fmt.Printf("%v\n", buf)
+		}
+	}
+
+	// ip address
+	if buf, err := ipmi.GetLanConfig(1, IPAddress); err != nil {
+		fmt.Printf("Failed to get LAN config: %v\n", err)
+	} else {
+		fmt.Printf("IP Address        : ")
+		if len(buf) == 6 {
+			fmt.Printf("%d.%d.%d.%d\n", buf[2], buf[3], buf[4], buf[5])
+		} else {
+			fmt.Printf("Unknown\n")
+		}
+	}
+
+	// MAC address
+	if buf, err := ipmi.GetLanConfig(1, MACAddress); err != nil {
+		fmt.Printf("Failed to get LAN config: %v\n", err)
+	} else {
+		fmt.Printf("MAC Address       : ")
+		if len(buf) == 8 {
+			fmt.Printf("%02x:%02x:%02x:%02x:%02x:%02x\n", buf[2], buf[3], buf[4], buf[5], buf[6], buf[7])
+		} else {
+			fmt.Printf("Unknown\n")
+		}
+	}
+}
+
+func sendRawCmd(cmds []string) {
+	ipmi, err := ipmi.Open(0)
+	if err != nil {
+		fmt.Printf("Failed to open ipmi device: %v\n", err)
```

Comment:
same thing here. You should log.Fatal(err). You might consider having this function return an error but for now log.Fatal is fine.